tests/helper/helper_oc.go

```diff
@@ -228,20 +228,64 @@ func (oc *OcRunner) SourceLocationBC(componentName string, appName string, proje
 	return sourceLocation
 }
 
-// ImportJavaIsToNspace import the openjdk image which is used for jars
-func (oc *OcRunner) ImportJavaIsToNspace(project string) {
-	// do nothing if running on OpenShiftCI
-	// java image is already present
-	val, ok := os.LookupEnv("CI")
-	if ok && val == "openshift" {
+// checkForImageStream checks if there is a ImageStram with name and tag in openshift namespace
+func (oc *OcRunner) checkForImageStream(name string, tag string) bool {
```

Comment:
Please include the namespace as an argument so that the function can also check the IS in a specified namespace.

Reply:
It is not needed, anywhere. The only place where we want to check is `openshift` namespace.

Reply:
ok, if later there will be a need then will expose that argument.